### DISCUSSION TITLE
Who create IPA for windows

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140936] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Send your explanations here, or even some documentation.

**Comment by EmersonNog** [comment]
It depends on what technology you are going to use.

**Comment by EmersonNog** [comment, marked as answer]
The only way to generate an IPA from you windows machine is by using a CI/CD service like [Codemagic](https://codemagic.io/start/) using the steps described in this [article](https://medium.com/flutter-community/developing-and-debugging-flutter-apps-for-ios-without-a-mac-8d362a8ec667):

link your repository (Github, BitBucket , or Gitlab).
from settings > build for platforms > IOS
change the .app file you received to .zip and extract it.
You will find a file called Runner.app, put in a folder and compress it back.
change the extension from .zip to .ipa.

And here's your first IOS build without a Mac device.

**Comment by LiteBrite82** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
